src/IROperator.h

```diff
@@ -1375,6 +1379,12 @@ Expr likely(Expr e);
  * found in an innermost loop. */
 Expr likely_if_innermost(Expr e);
 
+/** Expressions tagged with this intrinsic are suggestions that
+ * vectorization of loops with guard ifs should be implemented with
+ * non-faulting predicated loads and stores, instead of scalarizing
+ * an if statement. */
+Expr predicate(Expr e);
```

Comment:
Shouldn't this be in the internal namespace?

Reply:
I didn't realize likely_if_innermost wasn't in the internal namespace!